kernel/src/misc/wm/wm.c

```diff
@@ -469,6 +490,9 @@ void wm_mouse_callback(mouse_t raw_curr) {
     // Move the cursor
     float dx = (raw_curr.x - raw_prev.x)*sens;
     float dy = (raw_curr.y - raw_prev.y)*sens;
+    // store the mouse event for inclusion in the ringbuffer
+    wm_event_t drag_ev;
+    wm_event_t uc_ev;
```

Comment:
Don't declare those here, declare them in the innermost scope where they are used. Use unabbreviated names, `drag_event`, `move_event`, or just `event` (being declared close to usage gives enough context imo)